Introduction to Kotlin Multiplatform/04_js.md

```diff
@@ -64,20 +64,20 @@ with the following contents
 ```html
 <html lang="en">
 <body>
-<div id="app">Loading...</div>
-<script src="kotlin-fractals2-handson.js"></script>
+  <div id="app">Loading...</div>
+  <script src="kotlin-fractals2-handson.js"></script>
 </body>
 </html>
 ```
-We may include CSS styles to the HTML file to make it look
+We may include CSS styles to the HTML file to make the page in a browser look
```

Comment:
We can include CSS to the HTML file to make the page look better in the browser.